frontend/src/mixins/campRoleMixin.js

```diff
@@ -1,6 +1,6 @@
 export const campRoleMixin = {
   data: () => ({
-    loading: true
+    campRoleMixin_loading: true
```

Comment:
Ist ESLint happy mit underscore in data properties?

Reply:
Seems like, ja. Any other suggestion? Hat grad so schön als Prefix ausgesehen